scouting.py

```diff
@@ -15,12 +15,12 @@ def adder_page():
 
         try:
             team = int(request.form["team"])
-        except:
+        except ValueError:
             errors += "<p>{!r} is not a number.</p>\n".format(request.form["team"])
 
         try:
             game = int(request.form["game"])
-        except:
+        except ValueError:
             errors += "<p>{!r} is not a number.</p>\n".format(request.form["game"])
```

Comment:
These two are very similar - extract the common part to a function.
Additionally, you rely on exceptions here, but that's really not needed. The possible errors are either `KeyError` (`request.form` doesn't contain the accessed key) or `ValueError` (the associated value cannot be cast to a simple (base 10) integer).
You can (and should) handle these edge cases beforehand, instead of relying on exceptions.
I'd also pass around the value, instead of mutating `errors`.